packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -20,12 +20,12 @@ const unweakable = new WeakSet();
  *
  * @param {number} size  Maximum number of entries to keep in the cache before
  *    starting to throw them away.
- * @param {(baseRef: string) => Object} fetch  Function to retrieve an
+ * @param {(baseRef: string) => object} fetch  Function to retrieve an
  *    object's raw state from the store by its baseRef
- * @param {(baseRef: string, rawState: Object) => void} store  Function to
+ * @param {(baseRef: string, rawState: object) => void} store  Function to
```

Comment:
Here is potentially a place where it'd revert back to `any`

Reply:
It was already `any`. That's my point in the statement,
> Though it appears to do the same for Object so not relevant to this change?

`Object`
<img width="540" alt="Screen Shot 2022-05-03 at 12 19 02 PM" src="https://user-images.githubusercontent.com/21505/166534834-7fb3d1c5-3d92-4023-89aa-41812a821a2f.png">


`object`
<img width="535" alt="Screen Shot 2022-05-03 at 12 18 52 PM" src="https://user-images.githubusercontent.com/21505/166535197-9020566b-d9bb-4d25-8ea9-e790addf8c17.png">